PytonChallenge.NoaAizen

```diff
@@ -0,0 +1,13 @@
+file = open("pizza.txt", "r")
+data = file.read().split()
+print(data)
+dict={}
+for i in data:
```

Comment:
variable names should be clear and meaningful. in python it is not common to use 'i' as index var